### PR TITLE
Angus/readme updates

### LINKED ISSUE #7: Improve documentation for running after NPM install
Currently documentation explains how to run after cloning the repo. Should add instructions on installing from NPM and running

### LINKED ISSUE #24: Issue with logging in to dashboard with firefox
After the dashboard changes, users can no longer log in to the server using firefox

### PR DESCRIPTION
**Pull request by veggiesaurus**
adjustments to the readme

closes #7

### CODE REVIEW

**Review by ajm-ska — APPROVED** [review]
Looks OK to me. 

One thing, should the link to the "config documentation" show the compiled HTML? At the moment it links to the raw .html file. That might not be necessary as the .html file looks complicated and not intended for a user to read.
Could we host the compiled version on cartavis.github.io? Although I know it would be difficult to keep it up-to-date that way.

**Comment by veggiesaurus** [comment]
> Looks OK to me. 
> 
> One thing, should the link to the "config documentation" show the compiled HTML? At the moment it links to the raw .html file. That might not be necessary as the .html file looks complicated and not intended for a user to read.
> Could we host the compiled version on cartavis.github.io? Although I know it would be difficult to keep it up-to-date that way.

Not sure what you mean by compiled HTML... But I see the issue you raise. We could host a compiled version elsewhere o guess

**Comment by ajm-ska** [comment]
OK. I'm sorry. I'm using the wrong term to describe it. I mean we probably shouldn't link that file in the README because it isn't user readable. When I click the link it appears like this to me:
Screenshot 2020-09-18 at 16 36 56
By "compiled HTML", I meant how it looks when I use a web browser to view the file after cloning the repository:
Screenshot 2020-09-18 at 16 42 16

**Comment by veggiesaurus** [comment]
> OK. I'm sorry. I'm using the wrong term to describe it. I mean we probably shouldn't link that file in the README because it isn't user readable. When I click the link it appears like this to me:
> Screenshot 2020-09-18 at 16 36 56
> By "compiled HTML", I meant how it looks when I use a web browser to view the file after cloning the repository:
> Screenshot 2020-09-18 at 16 42 16

Right. I think perhaps the easiest solution is to just remove the link in the readme, and specify the path itself: 

> Additional details can be found in the auto-generated config documentation in the `docs` folder

File: README.md [hunk 1]
```diff
@@ -10,12 +10,18 @@ The CARTA server provides a simple dashboard which authenticates users and allow
...
+To allow the server to serve CARTA sessions, you must give it access to an executable CARTA backend, which can be either a compiled executable or a container. If you want to use a non-standard version of the CARTA frontend, you must also build it, and adjust the server configuration to point to it. You should use the `v1.4.0` tag of [`carta-backend`](https://github.com/CARTAvis/carta-backend).
 
 By default, the server runs on port 8000. It should be run behind a proxy, so that it can be accessed via HTTP and HTTPS. 
 
 MongoDB is required for storing user preferences, layouts and (in the near future) server metrics. You also need a working [NodeJS LTS](https://github.com/nvm-sh/nvm#long-term-support) installation with NPM. Use `npm install` to install all Node dependencies.
 
+## Installation
+
+The CARTA server can be installed from NPM by simply running `npm install -g carta-node-server`. To run the server, run `carta-node-server`.
```
Comment (confluence) [inline README.md]: "You can install the CARTA server from NPM by running..."

File: README.md [hunk 2]
```diff
@@ -34,6 +40,8 @@ Server configuration is handled by a configuration file in JSON format, adhering
 
 For external authentication systems, you may need to translate a unique ID (such as email or username) from the authenticated user information to the system user. You can do this by providing a [user lookup table](config/usertable.txt.stub), which is watched by the server and reloaded whenever it is updated.
 
+The server dashboard's appearance can be altered by adjusting the `dashboard` field in the config file. You can change the banner image and background, and add login instructions or institutional notices.
```
Comment (confluence) [inline README.md]: "You can alter the server's dashboard appearance by..."

**Review by confluence — CHANGES_REQUESTED** [review]

**Review by confluence — APPROVED** [review]

**Pull request merged**